New package: BrowserStackInc.Requestly version 25.10.29

almusavvir opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/312422)

almusavvir committed:
New package: BrowserStackInc.Requestly version 25.10.29

almusavvir commented:
@microsoft-github-policy-service agree

stephengillie commented:
Manual Validation ended with: 
> 2025-11-13 13:44:13.794 [CLI ] Package hash verification failed. SHA256 in manifest [46f1f2c5b96ebd1307584f39b4159cd4e801513a37e35f42203797e7f5351a26] does not match download [a0a498dd152b7c11a5de7f22e0cf9e34ab6d04fea34a128e3c0c0420079cf336] with file size [98809968] and content type [application/octet-stream]

(Automated response - build 1149.)

almusavvir committed:
Updated Requestly to version 25.11.13. Updated installer URL (now hardcoded). Bumped Packageversion

almusavvir commented:
Closing this PR to resubmit with updated manifests reflecting the latest installer release.

Corrected SHA256 hash

Updated InstallerUrl to official source

Bumped version in manifests

Will open a cleaner PR shortly to ensure clarity and smooth review.
